subsys/net/l2/Kconfig

```diff
@@ -72,6 +72,22 @@ source "subsys/net/l2/ppp/Kconfig"
 
 source "subsys/net/l2/ieee802154/Kconfig"
 
+config IEEE802154_NET_IF_NO_AUTO_START
+	bool "Disable 802.15.4 interface auto-start"
+	default n
+	depends on NET_L2_OPENTHREAD || NET_L2_IEEE802154
+
```

Comment:
Usually we do not put empty line before the help text

Reply:
done